src/puptoo/app.py

```diff
@@ -179,6 +190,8 @@ def handle_message(msg):
     if facts:
         facts["stale_timestamp"] = get_staletime()
         facts["reporter"] = "puptoo"
+        if facts.get("system_profile"):
```

Comment:
@SteveHNH  is it possible for "facts" or "system_profile" to be empty?
We want the key to *always* be added when cert auth is in use.